molecule/templates/vagrantfile.j2

```diff
@@ -40,16 +40,20 @@ Vagrant.configure('2') do |config|
     {%- if provider.type is defined and provider.type == 'virtualbox' and provider.name == current_provider %}
   config.vm.provider 'virtualbox' do |vb|
     {%- if provider.options %}
-      {%- for k, v in provider.options.iteritems()|sort %}
+      {%- for k, v in provider.options.iteritems() | sort %}
+        {%- if v == true or v == false %}
```

Comment:
Perhaps use "... == 'true' or v == 'false'" to make it clear that you are doing a *string* comparison?